apps/editors/templates/editors/emails/unlisted_to_sandbox.ltxt

```diff
@@ -0,0 +1,12 @@
+{% extends "editors/emails/base.ltxt" %}{% block content %}
+Your add-on, {{ name }} {{ number }}, didn't pass review and therefore won't be signed.
```

Comment:
There aren't any that I'm aware of. From what I understand, at that point it's clear that the add-on is a malware. Otherwise there would have been some back and forth between the editor and the dev.

The only "next step" is to submit a new version.